tests/ImageSharp.Tests/TestUtilities/FeatureTesting/FeatureTestRunner.cs

```diff
@@ -62,7 +62,7 @@ public static T Deserialize<T>(string value)
             ProcessStartInfo processStartInfo = new();
             if (intrinsic.Key != HwIntrinsics.AllowAll)
             {
-                processStartInfo.Environment[$"COMPlus_{intrinsic.Value}"] = "0";
+                processStartInfo.Environment[$"DOTNET_{intrinsic.Value}"] = "0";
```

Comment:
`COMPlus` prefix changed with .NET 6 to be `DOTNET`.
The COMPlus is still supported, but may get obsolete in future versions.

Reply:
Thanks for that. One of those things we'd have only found out once all our tests break.